### DISCUSSION TITLE
Have been Rejected for Github education for 9 Times. I have Done all The missing steps that I got after rejection but still the same

### DISCUSSION THREAD

**Discussion by ItzDc02** [community/community#128054] [GitHub Education]
### Hi everyone,

Hi 
I Applied for the GitHub education after I started doing my masters now. So I thought why not apply for it again but now I am getting instantly rejected. This didnt happen back when I had applied for it before. Back then it was instantly approved but now well yeah.
Can someone help me out with this. And yes I made sure to try My ID, My Academic admission form. I dont have any other documents beside my Course Payment Slips but I dont think I am comfortable enough to share that. And no my Email doesnt have any rejection reason anymore its just blank

**Comment by samus-aran** [comment]
Hi @ItzDc02 👋 

In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.
- Also ensure you take a live capture of your ID not a screenshot.

**Reply by ItzDc02** [reply]
How do I take the picture with my front camera exactly? The option doesn't work on the website with the back camera
